doc/changelog/08-vernac-commands-and-options/18880-deprecate_let_variable_outside_section.rst

```diff
@@ -0,0 +1,6 @@
+- **Changed:**
+  Clarify the warning about use of :cmd:`Let`, :cmd:`Variable`,
+  :cmd:`Hypothesis` and :cmd:`Context` outside sections and turn it as
+  error by default
+  (`#18880 <https://github.com/coq/coq/pull/18880>`_,
```

Comment:
I see several small changes in `vernacentries.ml` (calls to `enforce_locality_exp`) such as for Axiom, Definition, Fixpoint, CoFixpoint etc.  If their behavior has changed, maybe also mention them here.

Reply:
The only missing ones are `Let Fixpoint` and `Le Cofixpoint` but those are only variants of `Let`.